tests/ts_util.erl

```diff
@@ -504,7 +504,7 @@ get_optional(N, X) ->
     end.
 
 
--define(DELTA, 1.0e-10).
```

Comment:
Yes, the numbers between the two standard deviation calculations are close but because they are not an exact match I wanted to tighten up the range.  Sadly comparing floats is not an exact science.